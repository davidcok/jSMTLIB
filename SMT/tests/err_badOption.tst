; tests whether we get a message from a badly formed SMT option
(get-option zzz)
(get-option)
(get-option :zzz :www)
