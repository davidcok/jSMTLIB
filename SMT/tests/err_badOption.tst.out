(get-option zzz)
            ^^^
(error "Expected a keyword (beginning with a colon) here, instead of a symbol")
(get-option)
           ^
(error "Expected a keyword (beginning with a colon) here, instead of a )")
(get-option :zzz :www)
                 ^^^^
(error "Too many arguments or extraneous material after the command or missing right parenthesis")
